packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -85,6 +95,37 @@ function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
       </Flex>
     );
 
+    // read only    
+    if (isReadOnly && readOnlyText || readOnlyText === '') {
```

Comment:
I think this should be
```
 if (isReadOnly && (readOnlyText || readOnlyText === '')) {
```

found looking at the tests that failed, the add person dialog was displaying in readonly